tests/fs-test.rs

```diff
@@ -39,7 +39,7 @@ impl AsStr for &str {
 
 /// Gets a "base" guest and host path, only useful for UhyveFileMap tests.
 fn get_default_paths() -> (PathBuf, PathBuf) {
-	let guest_dir_path: PathBuf = PathBuf::from("/root/");
+	let guest_dir_path: PathBuf = PathBuf::from("/");
```

Comment:
Would it be feasible to include overshadowing as part of the test coverage? https://github.com/hermit-os/kernel/issues/1572